### ISSUE TITLE
render highway=pedestrian at the same zoom levels as highway=residential

### ISSUE THREAD

**Issue by dieterdreist** [gravitystorm/openstreetmap-carto#4510]
### Expected behavior
highway=pedestrian is rendered at the same zoom levels as highway=residential and the street grid remains recognizable and without holes in z12 and z13.

### Actual behavior
residential and even tracks remain visible in z13 but the highway=pedestrian ways, often main roads in historic city and town centres, are hidden, thereby promoting a biased map image, suitable only for car navigation, and compromising the urban shape and road structure.

#### Tübingen
https://www.openstreetmap.org/#map=13/48.5190/9.0570
![z13](https://user-images.githubusercontent.com/581288/150564806-04eb4455-efb0-4510-824f-2bc67f7ce8b5.png)
![z14](https://user-images.githubusercontent.com/581288/150564815-a08a6434-797e-46d5-b54f-1830e45b73f4.png)
#### Rome
https://www.openstreetmap.org/#map=13/41.8988/12.4880
![z13](https://user-images.githubusercontent.com/581288/150565503-2fdfa80e-da5b-43ca-a192-fccd94d6bffb.png)
![z14](https://user-images.githubusercontent.com/581288/150565506-ee55d383-713e-4a93-9b0f-bd19f859c96b.png)

**Comment by matkoniecz** [comment]
> biased map image, suitable only for car navigation, and compromising the urban shape and road structure.

I must say that person who authored it is really quite far from being car centrist.

The reason for that is at that scale of map foot/bicycle navigation becomes far less relevant and displaying of this roads obscures rather than illuminates.

**Comment by boothym** [comment]
Suppose the best thing to progress this issue would be some test renderings of some city centres for a direct comparison at z13?

**Comment by matkoniecz** [comment]
Yes, exactly. It requires working setup but the change should be really simple and easy to locate.
